src/main/kotlin/racingcar/domain/Race.kt

```diff
@@ -0,0 +1,25 @@
+package racingcar.domain
+
+class Race {
+    private val cars: List<Car>
+
+    constructor(carCount: Int) {
+        this.cars = (0 until carCount).map { Car() }
+    }
+
+    constructor(cars: List<Car>) {
+        this.cars = cars
+    }
```

Comment:
아래와 같은 주 생성자(primary constructor)를 만든 후에 자기 생성자 호출하는 `this()`를 사용하면 어떨까요?

```suggestion
class Race(private val cars: List<Car>) {
    constructor(carCount: Int) : this((0 until carCount).map { Car() })
```

Reply:
너무 좋습니다 :) 바로 적용하겠습니다! @wotjd243